.travis.yml

```diff
@@ -15,6 +15,7 @@ php:
 
 # faster builds on new travis setup not using sudo
 sudo: false
+dist: trusty
```

Comment:
It's better to specify trusty for specific PHP versions: https://github.com/yiisoft/yii2/blob/master/.travis.yml#L75